File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/factory/transportmethod/Bike.java
package com.bigboxcode.designpattern.factory.transportmethod;

public class Bike implements Transport {
    @Override
    public void start() {
        System.out.println("Bike started");
    }

    @Override
    public void stop() {
        System.out.println("Bike Stopped");
    }

    @Override
    public void repair() {
        System.out.println("Bike Repair");
    }
}

File: design-patterns/php/factory/transportmethod/Bike.php
<?php
// Bike.php

namespace BigBoxCode\DesignPattern\Factory\TransportMethod;

class Bike implements Transport {
    public function start() {
        echo "Bike started\n";
    }

    public function stop() {
        echo "Bike Stopped\n";
    }

    public function repair() {
        echo "Bike Repair\n";
    }
}